flask_security/datastore.py

```diff
@@ -124,7 +124,7 @@ def _prepare_role_modify_args(self, user, role):
         return user, role
 
     def _prepare_create_user_args(self, **kwargs):
-        kwargs.setdefault('active', True)
```

Comment:
@eliaperantoni could we simply remove the line and rely on default in user defined datastore?